### ISSUE TITLE
geog request: Myanmar

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#6007]
@dustymc can you reload **Myanmar** from service? I dont know why Kayah State is missing for Phyllis because it's there in the shapefile (and the other missing states) 

Same with reloading **Sri Lanka**-- if we stick with adm_1 then we should have 25 districts (all included in GADM v4.1)

_Originally posted by @mkoo in https://github.com/ArctosDB/arctos/issues/5654#issuecomment-1469272445_

**Comment by dustymc** [comment]
Here's CSV:

[temp_pre_geog(1).csv](https://github.com/ArctosDB/arctos/files/10981110/temp_pre_geog.1.csv)

Please review and check carefully against what's in Arctos (which is now GADMified).

**Comment by mkoo** [comment]
Why does your list have 6 and GADM the geodb have 15 adm_1 records?

**Comment by dustymc** [comment]
https://arctos.database.museum/place.cfm?sch=geog&country=Myanmar

11 exist (minus one for the country - there's a duplicate??)

**Comment by mkoo** [comment]
Actually better yet-- is there a check we can run? Create an index of counts and then run an internal tally to see if there's the correct number of states per country.

**Comment by dustymc** [comment]
taking this to https://github.com/ArctosDB/arctos/issues/5654

**Comment by mkoo** [comment]
ok priorities -- can you at least reload **Myanmar** and **Sri Lanka** which we know are missing some adm_1's

The comment about running checks is not urgent, just a comment about checking the GADM scripts since it seems to have missing out on importing stuff that is present in the GADM db. 

This issue can be closed once we confirm that those two countries have all their states/provinces/districts

**Comment by dustymc** [comment]
Myanmar done using csv from ^^ up there.

Gotcha, the last (first? however ya wanna look at it..) pass I just added GADM data to what was in Arctos, and standardized enough to do that and no more. We need another pass involving full GADMization (which is done for these two, and several other recents) - assuming we're still committed to finding SOME way to live with https://github.com/ArctosDB/internal/issues/222.

No, I'm allergic to XLS, and (at least as far as I can see, which I sometimes suspect isn't very far!) things have been a lot better since we stopped using WKT. (I'm using shp2pgsql on shapefiles that I pull directly in from gadm now.)
